src/ConfigPostProcessor.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-zendframework-bridge for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-zendframework-bridge/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-zendframework-bridge/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\ZendFrameworkBridge;
+
+class ConfigPostProcessor
+{
+    /** @var Replacements */
+    private $replacements;
+
+    public function __construct()
+    {
+        $this->replacements = new Replacements([
+            // Since we're rewriting at the PHP level, and not the raw
+            // strings from reading file contents, we need to add this
+            // one.
+            'zend-expressive' => 'expressive',
```

Comment:
That's a good question.

The problem is that we need to rewrite the config key `zend-expressive` to `expressive`.

However, there may _also_ be keys _starting_ with `zend-expressive` that would get rewritten in that case.

I can see the following solutions, and each poses... challenges:

- Leave as I've written it, and potentially have it rewrite keys we do not control.
- Add a mechanism for doing replacements on exact matches (pseudo-code: `exactMatchExists($key) ? exactReplace($key) : replacements($key)`. This could become a potential maintenance issue if we start identifying more such cases.

I think I'll go ahead with the second option (there aren't many, based on what I was identifying when we created the original lists for laminas-migration).